compiler/config.py

```diff
@@ -15,13 +15,16 @@
 else:
     PASH_TOP = subprocess.run(GIT_TOP_CMD, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True).stdout.rstrip()
 
+PARSER_BINARY = os.path.join(PASH_TOP, "compiler/parser/parse_to_json.native")
+PRINTER_BINARY = os.path.join(PASH_TOP, "compiler/parser/json_to_shell.native")
+
 PYTHON_VERSION = "python3"
 PLANNER_EXECUTABLE = os.path.join(PASH_TOP, "compiler/pash_runtime.py")
 RUNTIME_EXECUTABLE = os.path.join(PASH_TOP, "compiler/pash_runtime.sh")
 
 ## This is set in pash.py and pash_runtime.py accordingly.
 ## In both cases the setting is different.
-PASH_TMP_PREFIX = None
+PASH_TMP_PREFIX = "pash_"
```

Comment:
Is this necessary here? I think that it is set properly in `pash.py` and `pash_runtime.py`.